src/app/games/gos/tri-go/TriGoTutorial.ts

```diff
@@ -0,0 +1,81 @@
+import { GoMove } from 'src/app/games/gos/GoMove';
+import { GoState } from 'src/app/games/gos/GoState';
+import { GoPiece } from '../GoPiece';
+import { MGPOptional } from '@everyboard/lib';
+import { Tutorial, TutorialStep } from '../../../components/wrapper-components/tutorial-game-wrapper/TutorialStep';
+import { PlayerNumberMap } from 'src/app/jscaip/PlayerMap';
+import { TutorialStepMessage } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStepMessage';
+import { TriGoConfig, TriGoRules } from './TriGoRules';
+
+const X: GoPiece = GoPiece.LIGHT;
+const O: GoPiece = GoPiece.DARK;
+const k: GoPiece = GoPiece.DEAD_LIGHT;
+const w: GoPiece = GoPiece.LIGHT_TERRITORY;
+const b: GoPiece = GoPiece.DARK_TERRITORY;
+const _: GoPiece = GoPiece.EMPTY;
+const N: GoPiece = GoPiece.UNREACHABLE;
+
+const defaultConfig: MGPOptional<TriGoConfig> = TriGoRules.get().getDefaultRulesConfig();
+
+export class TriGoTutorial extends Tutorial {
+
+    public tutorial: TutorialStep[] = [
+        TutorialStep.informational(
+            $localize`Preliminary information`,
+            $localize`The game of TriGo is a triangular adaptation of the game of Go. Go is present on the website, you can go learn it here. This tutorial will only review the small differences that this experimental adaptation induced.`,
+            TriGoRules.get().getInitialState(defaultConfig),
+        ),
+        TutorialStep.informational(
+            $localize`Freedom (1/4)`,
+            $localize`Since it is triangular, pieces have only one freedom in the corners.`,
+            new GoState([
+                [N, N, N, N, N, N, _, N, N, N, N, N, N],
+                [N, N, N, N, N, _, _, _, N, N, N, N, N],
+                [N, N, N, N, _, _, _, _, _, N, N, N, N],
+                [N, N, N, _, _, _, _, _, _, _, N, N, N],
+                [N, N, _, _, _, _, _, _, _, _, _, N, N],
+                [N, _, _, _, _, _, _, _, _, _, _, _, N],
+                [O, _, _, _, _, _, _, _, _, _, _, _, X],
+            ], PlayerNumberMap.of(0, 0), 0, MGPOptional.empty(), 'PLAYING'),
+        ),
+        TutorialStep.informational(
+            $localize`Freedom (2/4)`,
+            $localize`Since it is triangular, pieces have only two on the edges (when they touch the edge with their corner only, their not really on the edges).`,
```

Comment:
two freedoms*
pas parenthèser de phrase, c'est du mauvais style.
"Since it is triangular, pieces have only two freedoms on the edges. When a piece touches the edges with its corner only, it is not really on the edge"

Reply:
waw, j'ai déjà plurielé et déparenthesé ! traufaur !